packages/ui/src/pages/AboutPage.js

```diff
@@ -127,7 +127,8 @@ class AboutPage extends Component {
               </p>
```

Comment:
Was this caused by the linter/formater?

Reply:
Yep, didn't intentionally touch anything in `ui`.